arbnode/seq_coordinator.go

```diff
@@ -508,12 +454,12 @@ func (c *SeqCoordinator) update(ctx context.Context) time.Duration {
 		}
```

Comment:
lines 410..449 have a loop reading messages into an array. I think it could be turned into a member function of RedisCoordinator. Something like `GetMessages(start, end) []arbstate.MessageWithMetaData`

Reply:
will handle in future PR